src/routes.js

```diff
@@ -17,7 +16,7 @@ const router = Router();
 /**
  * GET /api/swagger.json.
  */
-router.get('/swagger.json', (req, res) => {
+router.get('/swagger.json', (_, res) => {
```

Comment:
it's okay to keep req

Reply:
It wasn't being used so I'm trying to follow this convention for the unused variables.